lib/hcrypto/Makefile.am

```diff
@@ -2,7 +2,8 @@
 
 include $(top_srcdir)/Makefile.am.common
 
-AM_CPPFLAGS += -I$(srcdir)/libtommath -DUSE_HCRYPTO_LTM=1
+AM_CPPFLAGS += -I$(top_srcdir)/lib/base -I$(top_srcdir)/lib/hx509/ref \
```

Comment:
Maybe `-I$(top_srcdir)/lib/hx509`, without the `/ref`, to make sure that `#include <ref/pkcs11.h>` means that one, compared with `#include <pkcs11.h>` perhaps getting a system one?